CHANGELOG.md

```diff
@@ -2,6 +2,41 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 18 end 2015-01-26
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+18+Ending+Jan+26%2C+2015%22+label%3Aenhancement)
+
+- Providers
+  - Added refresh status and errors, viewable on Provider Summary Page
+  - OpenStack Host collection and metrics
+  - Amazon EC2:  Added C4 instance types
+- Rest API: VM start, stop, suspend, delete actions added.
+- UI
+ - Multi-character set language support
+ - Cloud Stacks summary and list views
+- Service Dialogs: Dynamic field support for text boxes and text area boxes
+- IPv6:  Pass IPv6 safe address to the socket layer
```

Comment:
IPv6: Ensure an IPv6 literal address, such as "[::1]", passed to the Excon gem removes the square brackets [],  when establishing a Socket connection to the address, "::1".

@mfeifer better?